Modules/LearningSequence/classes/Player/ilKioskPageRenderer.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * Class ilKioskPageRenderer
+ * taken from former PR covering the page-ui-element.
+ */
+class ilKioskPageRenderer
+{
+
+	public function __construct($tpl)
+	{
+		$this->ilTpl = $tpl;
+	}
+
+	public function render(string $content)
+	{
+		$tpl = new \ilTemplate("tpl.kioskpage.html", true, true, 'Modules/LearningSequence');
+		$tpl->setVariable('CONTENT', $content);
+		$tpl = $this->setHeaderVars($tpl, $this->ilTpl);
+		return $tpl->get();
+	}
+
+	protected function setHeaderVars($tpl, $il_tpl) {
```

Comment:
kann man in dieser funtion die benamsung ein wenig ändern? das sieht nicht nach code von uns aus.
auch kommt ein leerzeichen hinter if/foreach/while usw.